src/app/clusters/on-off-server/on-off-server.h

```diff
@@ -17,19 +17,83 @@
 
 #pragma once
 
+#include <app-common/zap-generated/cluster-objects.h>
 #include <app/util/af-types.h>
 #include <app/util/basic-types.h>
 
-/** @brief On/off Cluster Set Value
- *
- * This function is called when the on/off value needs to be set, either through
- * normal channels or as a result of a level change.
+/**********************************************************
+ * Defines and Macros
+ *********************************************************/
+
+#define UPDATE_TIME_MS 100
```

Comment:
nit: should these be C++ constants (i.e. `static constexpr`)? That provides a bit of  type checking

Reply:
done